src/js/Framework/Theme.js

```diff
@@ -0,0 +1,76 @@
+import {Cookies} from './Cookies'
+const cookies = new Cookies()
+
+export class Theme {
+  constructor () {
+    this.getTheme()
+    this.themeCookie()
+    this.initChangeTheme()
+    this.doNotHideDropdown()
+  }
+
+  initChangeTheme () {
+    // on click, do changeTheme
+    $(document).on('click', '[data-theme-toggler]', $.proxy(this.changeTheme, this))
+    $('[data-dropdown-user-wrapper]').on('hide.bs.dropdown', $.proxy(this.doNotHideDropdown, this))
+  }
+
+  getTheme () {
+    // if cookie is not yet set, get the theme of the device, default light
+    if (cookies.readCookie('theme') == null) {
+      if (window.matchMedia && window.matchMedia('(prefers-color-scheme: dark)').matches) {
+        cookies.setCookie('theme', 'dark')
+      } else {
+        cookies.setCookie('theme', 'light')
+      }
+    }
+  }
+
+
+  changeTheme (e) {
+    e.preventDefault()
+    // read cookie --> currentTheme & themeToBe setten
+    let $themeToBe = 'dark'
+
+    if (cookies.readCookie('theme') === 'dark') {
```

Comment:
Waarom lees je de checkbox value niet uit, dat is toch het punt van de switch?

Reply:
Ah ja, in het begin waren dat 2 buttons, vandaar dat het zo was, maar goeien bal dat dit aangepast kan worden